main.go

```diff
@@ -105,6 +114,9 @@ func execute(dbprovider string) {
 		cache.ResourceEventHandlerFuncs{
 			AddFunc: func(obj interface{}) {
 				db := obj.(*crd.Database)
+				if excluded(db, excludeNamespaces, includeNamespaces) {
+					return
```

Comment:
Perhaps log something to indicate that we are skipping this

Reply:
Actually logging is done in the excluded function. I chose that in order to have differing messages for either flag. Also alleviates duplication of the logging lines in the add/delete handlers.